openvox.gemspec

```diff
@@ -0,0 +1 @@
+puppet.gemspec
```

Comment:
I would prefer if if the openvox gemspec is the actual one, but that probably doesn't matter.

Reply:
I thought about that too with the eye on the future where `puppet.gemspec` may be an empty spec that only depends on `openvox`. We can still do that.

Reply:
Mhm true, we can always flip it.